manifests/remove_setuid.pp

```diff
@@ -17,7 +17,7 @@
     exec { "remove setuid from ${file}":
       path    => ['/usr/bin', '/usr/sbin'],
       command => "chmod ug-s ${file}",
-      onlyif  => "find ${file} -perm -4000",
+      onlyif  => "test `find ${file} -perm -4000 | wc -l` -gt 0",
```

Comment:
Since the code removes both setuid and setgid, the find command should find files that have both set, or only setuid, or only setgid set.

I tested a bit and this should work, but double check
`find ${file} -perm -4000 -o -perm -2000`

Was also just curious why the pipe to wc is needed, is the return from find kinda wonky in some cases?

Reply:
Yes, the find command is a bit weird on what it returns. Or at least I haven't understood it.

I'm not the best with setuid & setgid. Thanks for the tip so that we are returning both.

@bsper2 Should we rename this somehow to reflect that we are doing both? And/or split out setuid vs setgid?

Reply:
Yeah a rename might be nice to make it clear it does both. Splitting it out would probably be best since that makes it the most flexible, but I think a single class does does them both is ok. I would think the use case where there is a file we want to remove one but not the other would be pretty rare.